src/GameServer/MessageHandler/TargetedSkillHandlerPlugIn.cs

```diff
@@ -45,11 +45,6 @@ public virtual async ValueTask HandlePacketAsync(Player player, Memory<byte> pac
     /// <param name="targetId">The target identifier.</param>
     protected async ValueTask HandleAsync(Player player, ushort skillId, ushort targetId)
     {
-        if (player.SkillList is null || !player.SkillList.ContainsSkill(skillId))
```

Comment:
This check is already done in the targeted skill plugins (and the default/base plugin).  
Reason for removing this check is that the nova start skill is not registered in the user skills - only the nova end)